drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -445,35 +445,29 @@ local function migrate_light(driver, device, parent_device_id, hue_light_descrip
     )
   )
 
-  local profile_ref = "white"
-  if light_resource.color then
-    log.debug(string.format(
-      "Migrating Color Ambiance light %s", (device.label or device.id or "unknown_device")
-    ))
-    profile_ref =
-    "white-and-color-ambiance" -- all color light_resource products support `white` (dimming) and `ambiance` (color temp)
-  elseif light_resource.color_temperature then
-    log.debug(string.format(
-      "Migrating White Ambiance light %s", (device.label or device.id or "unknown_device")
-    ))
-    profile_ref = "white-ambiance" -- all color temp products support `white` (dimming)
-  elseif light_resource.dimming then
-    log.debug(string.format(
-      "Migrating White light %s", (device.label or device.id or "unknown_device")
-    ))
-    profile_ref = "white" -- `white` refers to dimmable and includes filament bulbs
-  else
-    log.warn(
-      string.format(
-        "light_resource [%s] with Hue resource ID [%s] does not seem to be A White/White-Ambiance/White-Color-Ambiance device, assigning to `white` as fallback"
-        , (device.label or device.id or "unknown device")
-        , light_resource.id
-      )
-    )
-  end
+  local dbg_table = {
+    bulb_label = (device.label or device.id or "unknown identifier"),
+    profile_supports = {
+      switch = device:supports_capability_by_id(capabilities.switch.ID, nil),
+      switchLevel = device:supports_capability_by_id(capabilities.switchLevel.ID, nil),
+      colorTemperature = device:supports_capability_by_id(capabilities.colorControl.ID, nil),
```

Comment:
Looks like you need to swap the colorTemperature and colorControl mapping here
```suggestion
      colorTemperature = device:supports_capability_by_id(capabilities.colorTemperature.ID, nil),
```